pkg/controller/account/account_controller.go

```diff
@@ -428,14 +428,8 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 
 		if err != nil {
 			var returnErr error
-			if aerr, ok := err.(awserr.Error); ok {
-				reqLogger.Error(returnErr,
-					fmt.Sprintf(`Unable to create AWS connection with SRE credentials, 
-						AWS Error Code: %s, 
-						AWS Error Message: %s`,
-						aerr.Code(),
-						aerr.Message()))
-			}
+			errMsg := `Unable to create AWS connection with SRE credentials`
+			controllerutils.LogAwsError(reqLogger, errMsg, returnErr, err)
```

Comment:
I think this would be better to collapse into
controllerutils.LogAwsError(reqLogger, "Unable to create AWS connection with SRE credentials", returnErr, err)

Reply:
Done